### ISSUE TITLE
Typescript kit not working after installing aurelia-dialog 0.5.4

### ISSUE THREAD

**Issue by dli04** [aurelia/dialog#86]
After downloading the typescript kit and adding in aurelia-dialog 0.5.4, the kit doesn't fire up correctly anymore. The aurelia-bootstrapper is not being loaded; even if I don't use the aurelia-dialog in the configuration setup. I tried the es-2016 kit and it is working fine with aurelia-dialog 0.5.4.

**Comment by PWKad** [comment]
Can you give an idea of any errors you are encountering?

**Comment by dli04** [comment]
I don't see any errors from the browser console and dos prompt. I use http-server for the web server and after the loading of the config.js and favorite icon, nothing else loads. It looks like the Aurelia-bootstrapper is never called.

**Comment by PWKad** [comment]
@dli04 I've just confirmed using the steps above that this is working as expected in the TypeScript skeleton.  Are you sure you have the latest of all of the releases?  Closing for now, please join our gitter channel if you need additional help.  I know that the TypeScript compiler will fail if you don't have the `.d.ts` file but that is distributed properly at the moment.

**Comment by dli04** [comment]
Thanks. I've downloaded the typescript skeleton and is now working properly.

However, the typescript kit from the http://aurelia.io/docs.html#/aurelia/framework/1.0.0-beta.1.1.2/doc/article/getting-started still doesn't work for me and I'm giving up on it. I did jspm clean and jspm update multiple times and below was as far as the loading could get to. No errors pop up in the shell or browser console.


shell> 
   http-server -o -p8091

Starting up http-server, serving ./
Available on:
  http:127.0.0.1:8091
Hit CTRL-C to stop the server
[Tue, 09 Feb 2016 19:37:35 GMT] "GET /" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.103 Safari/537.36"
[Tue, 09 Feb 2016 19:37:35 GMT] "GET /styles/styles.css" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.103 Safari/537.36"
[Tue, 09 Feb 2016 19:37:35 GMT] "GET /jspm_packages/system.js" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.103 Safari/537.36"
[Tue, 09 Feb 2016 19:37:35 GMT] "GET /config.js" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.103 Safari/537.36"
[Tue, 09 Feb 2016 19:37:35 GMT] "GET /favicon.ico" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.103 Safari/537.36"
[Tue, 09 Feb 2016 19:37:55 GMT] "GET /jspm_packages/system.js.map" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.103 Safari/537.36"
